packages/webcomponents/src/components/payment-method-form/payment-method-form.tsx

```diff
@@ -9,19 +9,21 @@ import {
 } from '@stencil/core';
 import iFrameResize from 'iframe-resizer/js/iframeResizer';
 import { MessageEventType } from './message-event-types';
-import { Theme } from './theme';
+import { Theme } from '../../utils/theme';
 import packageJson from '../../../package.json';
-import getComputedTheme from './get-computed-theme';
+import getComputedTheme from '../../utils/get-computed-theme';
 import { CreatePaymentMethodResponse } from './payment-method-responses';
-import { config } from '../../../config';
+import { PaymentMethodTypes } from '../../api';
+import { composeQueryParams } from '../../utils/utils';
+import { FrameCommunicationService } from '../../utils/frame-comunication-service';
 
 @Component({
   tag: 'justifi-payment-method-form',
   styleUrl: 'payment-method-form.css',
   shadow: false,
 })
 export class PaymentMethodForm {
-  @Prop() paymentMethodFormType: 'card' | 'bankAccount';
+  @Prop() paymentMethodFormType: PaymentMethodTypes.card | PaymentMethodTypes.bankAccount;
```

Comment:
could this be written as `@Prop() paymentMethodFormType: PaymentMethodTypes` ? 

instead of `.card | .bankAccount` ?